pennylane_lightning/src/simulators/lightning_kokkos/measurements/MeasuresFunctors.hpp

```diff
@@ -34,35 +34,9 @@ template <class PrecisionT> struct getProbFunctor {
 
     KOKKOS_INLINE_FUNCTION
     void operator()(const size_t k) const {
-        PrecisionT REAL = arr[k].real();
-        PrecisionT IMAG = arr[k].imag();
-        probability[k] = REAL * REAL + IMAG * IMAG;
-    }
-};
-
-/**
- *@brief Compute cumulative probability distribution from discrete probability
- *distribution and the cumulative probability distribution is then stored into
- *probabilities_.
- *
- *@param probabilities_ Discrete probability distribution.
- */
-template <class PrecisionT> struct getCDFFunctor {
```

Comment:
Was this not being used?

Reply:
Single use, so replaced by `KOKKOS_LAMBDA` function.